### PR TITLE
API Updates

### PR DESCRIPTION
**Pull request by armando-rodriguez-cko**
API Updates
- Add local_schemes and deprecate scheme_local
- Feat(issuing): simulate preauth auths & reversals
- Feat(issuing): simulate clearing

### CODE REVIEW

**Review by martinseco — CHANGES_REQUESTED** [review]
Left some comments

File: src/CheckoutSdk/Issuing/IssuingClient.CardTesting.cs [hunk 1]
```diff
@@ -19,5 +19,51 @@ public partial class IssuingClient
...
+                cancellationToken
+            );
+        }
+
+        public Task<CardAuthorizationReversalResponse> SimulateReversal(
+            string authorizationId,
+            CardAuthorizationReversalRequest cardAuthorizationReversalRequest,
+            CancellationToken cancellationToken = default)
+        {
+            CheckoutUtils.ValidateParams("authorizationId", authorizationId, "cardAuthorizationRequest",
```
[the remark this answers is not in the document]
Comment (armando-ibarra-cko) [reply src/CheckoutSdk/Issuing/IssuingClient.CardTesting.cs:59]: The name of the string should be "cardAuthorizationReversalRequest"

File: src/CheckoutSdk/Issuing/Testing/Requests/CardAuthorizationClearingRequest.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+namespace Checkout.Issuing.Testing.Requests
+{
+    public class CardAuthorizationClearingRequest
+    {
+        public int Amount { get; set; }
```
Comment (armando-ibarra-cko) [inline src/CheckoutSdk/Issuing/Testing/Requests/CardAuthorizationClearingRequest.cs:5]: primitives requires `?`

File: src/CheckoutSdk/Issuing/Testing/Requests/CardAuthorizationIncrementingRequest.cs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+namespace Checkout.Issuing.Testing.Requests
+{
+    public class CardAuthorizationIncrementingRequest
+    {
+        public int Amount { get; set; }
```
Comment (armando-ibarra-cko) [inline src/CheckoutSdk/Issuing/Testing/Requests/CardAuthorizationIncrementingRequest.cs:5]: same here

File: src/CheckoutSdk/Issuing/Testing/Requests/CardAuthorizationReversalRequest.cs [hunk 4]
```diff
@@ -0,0 +1,7 @@
+namespace Checkout.Issuing.Testing.Requests
+{
+    public class CardAuthorizationReversalRequest
+    {
+        public int Amount { get; set; }
```
Comment (armando-ibarra-cko) [inline src/CheckoutSdk/Issuing/Testing/Requests/CardAuthorizationReversalRequest.cs:5]: here

File: src/CheckoutSdk/Issuing/IssuingClient.CardTesting.cs [hunk 5]
```diff
@@ -19,5 +19,51 @@ public partial class IssuingClient
...
+            );
+        }
+
+        public Task<EmptyResponse> SimulateClearing(
+            string authorizationId,
+            CardClearingAuthorizationRequest cardClearingAuthorizationRequest,
+            CancellationToken cancellationToken = default
+        )
+        {
+            CheckoutUtils.ValidateParams("authorizationId", authorizationId, "cardAuthorizationClearingRequest",
```
Comment (martinseco) [inline src/CheckoutSdk/Issuing/IssuingClient.CardTesting.cs:44]: "cardAuthorizationClearingRequest" should be cardClearingAuthorizationRequest

**Review by martinseco — APPROVED** [review]
Looks good! Just one small comment before merging

**Pull request merged**